SignInCheckIn/SignInCheckIn/Controllers/ChildSigninController.cs

```diff
@@ -210,5 +226,24 @@ public IHttpActionResult ReverseSignin(int eventparticipantid)
                 }
             });
         }
+
+        private void PublishSignedInParticipantsToRooms(ParticipantEventMapDto participants)
+        {
+            // loop through rooms that need to have an update and push the update to them
+            var rooms = participants.Participants.Select(m => m.AssignedRoomId).Distinct();
+            var eventId = participants.CurrentEvent.EventId;
+            foreach (var room in rooms)
+            {
+                // ignores the site id if there is an event id so therefore we can put a random 0 here
+                var updatedParticipants = participants.Participants.Where(p => p.AssignedRoomId == room);
+
+                PublishToChannel(_context, new ChannelEvent
+                {
+                    ChannelName = $"{_applicationConfiguration.CheckinParticipantsChannel}{eventId}{room}",
```

Comment:
Moved the creation of the channel name to a function.  Did not add the (add/remove) name to a function as that was causing to many things to be added to MPAuth that would have required any Controller using MPAuth to need them.